code/modules/clothing/suits/wintercoats.dm

```diff
@@ -29,6 +31,42 @@
 		/obj/item/toy,
 	)
 
+	generate_hood_overlay()
+
+
+/obj/item/clothing/suit/hooded/wintercoat/examine(mob/user)
+	. = ..()
+
+	. += span_notice("<b>Alt-click</b> to [zipped ? "un" : ""]zip.")
+
+
+/obj/item/clothing/suit/hooded/wintercoat/AltClick(mob/user)
+	. = ..()
+
+	if (. == FALSE) // Direct check for FALSE, because that's the specific case we want to propagate, not just null.
+		return FALSE
+
+	zipped = !zipped
+	worn_icon_state = "[initial(icon_state)][zipped ? "_t" : ""]"
+	to_chat(user, span_notice("You [zipped ? "" : "un"]zip [src]."))
```

Comment:
Should be a balloon alert like "zipped" and "unzipped".

Reply:
Done.